specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -260,6 +260,9 @@ model AcsEmailDeliveryReportReceivedEventData {
   /** The recipient Email Address */
   recipient?: string;
```

Comment:
Is it correct that this is an optional field? Wouldn't an email always have a recipient? /cc @l0lawrence

Reply:
This should be required. One question, if we change this to non-optional what impact will this have in the sdk?

Reply:
We have been allowing such changes as there should be no customer-facing impact.

Reply:
What about the sender property? Should that also be required?